addons/website_event_track_quiz/views/event_leaderboard_templates.xml

```diff
@@ -6,13 +6,13 @@
         <div t-if="visitors" class="bg-light pt32 pb32">
             <t t-call="website_event_track_quiz.leaderboard_search_bar"/>
             <div class="container mt32">
-                <div class="row mb-3">
+                <div t-if="not search" class="row mb-3">
                     <div class="col-md-4 d-flex flex-grow-1" t-foreach="top3_visitors" t-as="visitor">
                         <t t-call="website_event_track_quiz.top3_visitor_card"></t>
                     </div>
                 </div>
                 <table class="table table-sm">
-                    <tr t-foreach="visitors" t-as="visitor" class="bg-white">
+                    <tr t-foreach="visitors" t-as="visitor" t-att-class="'alert-info' if visitor['is_current_visitor'] else 'bg-white'">
```

Comment:
Would be better to send the current_visitor to the templates values and check `visitor['visitor'] == current_visitor`

That way you don't pass an extra dict key for the entire leaderboard that is False 99% of the time.